app/components/travis-layout.js

```diff
@@ -1,3 +1,12 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 
-export default Component.extend({});
+export default Component.extend({
+  classNameBindings: ['layoutClass'],
+
+  layoutName: '',
+
+  layoutClass: computed(function () {
```

Comment:
Should this computed property depend on `layoutName`? Currently it will only calculate on first request and never update afterwards. Is it expected?

Reply:
You know, I had a weird feeling that I was forgetting something haha. Yes, I think it should depend on `layoutName`. I've made that change. Thanks for catching it! :smile: